### PR TITLE
Reduce coverage of glass walls from 90% to 25%

### LINKED ISSUE #51400: Glass walls still have 90% coverage
### Describe the bug

Glass walls, the ones that can be found at normal houses at least, have 90% coverage. This is higher than a window's coverage. It doesn't make any sense, glass doors have 0% coverage though.

### Steps To Reproduce

Just press X and aim at a glass wall, or crouch next to it.

### Expected behavior

It should have 0% coverage, just like the glass door.

### Screenshots

_No response_

### Versions and configuration

1fa1d84

### Additional context

If it's a frosted wall, can we change the name to "Frosted Glass Wall" to clear confusion?

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
Balance "Reduce coverage of glass walls from  90% to 25%"

#### Purpose of change
Fixes #51400 by reducing coverage of glass walls and alarmed glass walls as expected

#### Describe the solution
Change coverage of t_wall_glass and t_wall_glass_alarm to 25% as they appear to represent large windows in many mapgen files.

#### Describe alternatives you've considered
Splitting t_glass_wall into full length glass walls and large windows, with 0% coverage for the former and 25% for the latter, though this would require editing a large number of mapgen files.

#### Testing
Loaded into base game and tested

#### Additional context
glass wall fixed

### CODE REVIEW

File: data/json/furniture_and_terrain/terrain-walls.json [hunk 1]
```diff
@@ -986,7 +986,7 @@
     "symbol": "LINE_OXOX",
     "color": "light_cyan",
     "move_cost": 0,
-    "coverage": 90,
+    "coverage": 25,
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/terrain-walls.json:989]:
```suggestion
    "coverage": 30,
```
Changing them both to thirty to test if that will cause tests to fire.

File: data/json/furniture_and_terrain/terrain-walls.json [hunk 2]
```diff
@@ -1011,6 +1011,7 @@
     "symbol": "LINE_OXOX",
     "color": "light_cyan",
     "move_cost": 0,
+    "coverage": 25,
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/terrain-walls.json:1014]:
```suggestion
    "coverage": 30,
```

**Comment by Maleclypse** [comment]
> @Maleclypse Is this a general issue or have I done something to my repository that is causing the tests not to fire? Had this on a couple of PRs recently and wanted to check I haven't done anything wrong from my side...

I don’t think you’ve done anything wrong on your side. But if it happens again please ping me and I’ll do some more tests and try to find out how to help you. Sometimes it’s mostly a matter of submitting at a busy time and the tests time out before they ever start.

**Review by anothersimulacrum — APPROVED** [review]

**Comment by kevingranade** [comment]
Super weird, I'll keep an eye out, but I really have no idea what is causing the tests to not run. If it happens again I'll try pushing an empty commit and see if that resolves it.

**Pull request merged**